etc/skydive.yml.default

```diff
@@ -159,6 +160,9 @@ analyzer:
       - serviceentry
       - virtualservice
 
+    ovn:
+      # address: unix:///var/run/openvswitch/ovnnb_db.sock
```

Comment:
could be nice to add a comment to specify that it has to be the north bound db address